src/logic/documents/scorecards.js

```diff
@@ -329,7 +332,15 @@ const scorecard = ({
   printStations,
   scorecardPaperSize,
   featured = false,
+  language = 'en',
 }) => {
+  const defaultTranslation = translation('en');
+  const translationData = translation(language);
+
+  const t = stringKey => {
+    return translationData[stringKey] || defaultTranslation[stringKey];
+  };
```

Comment:
You can do something like this:

```suggestion
  const defaultTranslationData = translation("en");
  const translationData = translation(language);
  
  const t = (...keys) => {
    const [phrase, defaultPhrase] = keys.reduce(
      [translationData, defaultTranslationData],
      ([data1, data2]) => [data1[key], data2[key]],
    );
  
    return phrase || defaultPhrase;
  };
```

So that we can conveniently query nested keys, such as `t('eventName', eventId)` and fallbacks work at any level automatically.

Reply:
Seems nice but I can't make this work :/ 
It was a bit better when I replaced 
`      ([data1, data2]) => [data1[key], data2[key]],`
with 
`      ([data1, data2], key) => [data1[key], data2[key]],`

but I still get an error when I generate the scorecards:
![skeudenn](https://github.com/user-attachments/assets/3d4dabe5-78a6-44d6-b7d5-4d200cb21f2c)

Do you have any idea on how to fix this?

Reply:
Oh my bad, the argument order is different, I've updated the snippet above, let me know if that works!

Reply:
Thanks, it works great!